coalib/settings/Setting.py

```diff
@@ -89,6 +89,24 @@ def typed_ordered_dict(key_type, value_type, default):
         for key, value in OrderedDict(setting).items())
 
 
+def _path_worker(path: str, origin: str, glob_escape_origin: bool=True):
```

Comment:
I find the name `_path_worker` totally confusing and it doesn't tell me what it does